integration/raft/channel_participation_test.go

```diff
@@ -533,6 +640,21 @@ var _ = Describe("ChannelParticipation", func() {
 	})
 })
 
+func submitTxn(o *nwo.Orderer, peer *nwo.Peer, network *nwo.Network, orderers []*nwo.Orderer,
+	expectedBlkNum int, expectedChannelInfo channelparticipation.ChannelInfo) {
+	By("submitting a transaction to " + o.Name)
+	env := CreateBroadcastEnvelope(network, peer, expectedChannelInfo.Name, []byte("hello"))
+	resp, err := ordererclient.Broadcast(network, o, env)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(resp.Status).To(Equal(common.Status_SUCCESS))
+	expectedBlockNumPerChannel := map[string]int{"participation-trophy": expectedBlkNum}
```

Comment:
```suggestion
	expectedBlockNumPerChannel := map[string]int{expectedChannelInfo.Name: expectedBlkNum}
```
Nit but we can fix it the next time we add/update an integration test here.